lib/cli/cliHost.js

```diff
@@ -167,12 +167,14 @@ module.exports = class CLIHost extends CLICommand {
                     return void callback(30);
                 }
 
+                /*
+                // TODO: we allow migration to already existing host - if no problem occurs, remove lines after September 2020
```

Comment:
Is it possible that not checking this opens up the possibility of having two identically-named hosts?

Reply:
Also thought about this, and this could of course be a problem and I don't see a possibility to check it and still allow the migration.

Reply:
We could actively ask the user, but I am approx. 100 % sure somebody will still manage to have multihost with identical names. So better prevent it like before.. at least for now